src/Tools/dotnet-gcdump/CommandLine/CollectCommandHandler.cs

```diff
@@ -32,6 +32,12 @@ private static async Task<int> Collect(CancellationToken ct, IConsole console, i
         {
             try
             {
+                if (processId < 0)
+                {
+                    Console.Out.WriteLine($"The PID cannot be negative: {processId}");
```

Comment:
If someone entered the command `dotnet-gcdump collect`, that would print this error, right? I think it might be better to just put default PID as 0 and add "--process-id is required" to this error check.